index.d.ts

```diff
@@ -133,6 +133,38 @@ type ReaderRevenuePosition =
     | 'ampHeader'
     | 'ampFooter';
 
+interface CampaignsType {
+    id: string;
+    name: string;
+    rules: [];
+    priority: number;
+    activeFrom: number;
+    displayOnSensitive: boolean;
+    fields: {
+        formId: number;
+        callout: string;
+        _type: string;
+        description: string;
+        tagName: string;
+        formFields: CampaignsFeildType[];
+    };
+}
+
+interface CampaignsFeildType {
+    text_size?: number | string; // TODO: need to fix at source
```

Comment:
Is it better to make this fix first rather than move past it and potentially (lets face it, probably) never fix it?